<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slm_machineClass</class>
 <widget class="QMainWindow" name="slm_machineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>slm_machine</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/slm_resources.qrc">
    <normaloff>:/icons/SLM_Logo</normaloff>:/icons/SLM_Logo</iconset>
  </property>
  <property name="locale">
   <locale language="Turkish" country="Turkey"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="buddyList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addBuddyButton">
          <property name="text">
           <string>Add Buddy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeBuddyButton">
          <property name="text">
           <string>Remove Buddy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>224</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionEncryption_Key"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_SLM"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_SLM">
   <property name="text">
    <string>About SLM</string>
   </property>
  </action>
  <action name="actionEncryption_Key">
   <property name="text">
    <string>Encryption Key</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/slm_resources.qrc"/>
 </resources>
 <connections/>
</ui>
